skyvern/webeye/browser_factory.py

```diff
@@ -485,6 +485,8 @@ async def set_working_page(self, page: Page | None, index: int = 0) -> None:
                         self.browser_artifacts.video_artifacts[index].video_path = await page.video.path()
                 except asyncio.TimeoutError:
                     LOG.info("Timeout to get the page video, skip the exception")
+                except Exception:
```

Comment:
this is fine but can't this just be `if page.video is not None: self.browser_artifacts.video_artifacts[index].video_path = await page.video.path()`?